### DISCUSSION TITLE
Error in intellij ultimate version 2024.2.0.1 (242.20224.387)

### DISCUSSION THREAD

**Discussion by joosecj** [community/community#135799] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Plugin Copilot: 1.5.18.6517

Error Message:

`Unfortunately, GitHub Copilot is unsupported on this machine. Please make sure that Node.js is installed and that "node" is available in the PATH.`

Configured Node:

![image](https://github.com/user-attachments/assets/e0c1f17c-14c4-48a8-be44-ff3d045cbbd2)

**Comment by thomas-sickert** [comment, marked as answer]
Sorry for the friction, `1.5.19` has been pushed with a fix for this. Let us know if you see any additional issues.
